test/no-redundant-jsdoc/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+  "rulesDirectory": ["../../bin/rules"],
+  "rules": {
+    "no-redundant-jsdoc": true
+  }
+}
```

Comment:
Note: This test is a **lie**

( because the TS version in this repo isn't 4.0, but when it is, it will cover the change safely )